package.json

```diff
@@ -6,7 +6,7 @@
     "start": "npm run server & npm run watch",
     "watch": "watchify -o dist/bundle.js -v -d js/app.js",
     "server": "http-server -p 8282 -s",
-    "build": "browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > js/bundle.min.js"
+    "build": "mkdir -p dist && browserify . -t [envify --NODE_ENV production] | uglifyjs -cm > dist/bundle.js"
```

Comment:
`watch` builds dist/bundle.js and uglifyjs is attempting to build dist/bundle.min.js. An alternative possibly similar approach would be to have the bundle build into /js. This is how the template work that I used to start. This avoids the mkdir.

Would you like to try that? Or I'm happy to make these updates as well. Thanks again for the bug report.